assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -176,13 +190,15 @@ def compute_feature_importance(task_type, target_column, baseline_data, categori
     :type baseline_data: pandas.DataFrame
     :param categorical_features: The column names which are categorical in type
     :type categorical_features: list[string]
+    :param local_explanations: If local explanations are wanted
+    :type local_explanations: boolean
```

Comment:
nit: i would make the boolean name more obviously a boolean something like is_local_explanations_enabled but this is preference